### PR TITLE
add job that continuously builds and releases debian packages

### PR DESCRIPTION
**Pull request by mikedanese**
This change is [Reviewable](https://reviewable.io/reviews/kubernetes/test-infra/622)

### CODE REVIEW

File: jenkins/job-configs/kubernetes-jenkins/kubernetes-debian-build.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        external-deletion-command: 'sudo rm -rf %s'
+
+- project:
+    name: kubernetes-debian-builds
+    build:
+    - 'unstable':
+         branch: 'master'
+         timeout: 20
+         job-env: |
+           export DEB_CHANEL="unstable"
```
Comment (spxtr) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-debian-build.yaml]: channel?

**Comment by spxtr** [comment]
If it's one job, why use a job template?

**Comment by mikedanese** [comment]
@spxtr I want to add one for stable shortly.

File: jenkins/job-configs/kubernetes-jenkins/kubernetes-debian-build.yaml (hunk 1 above)
Comment (mikedanese) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-debian-build.yaml]: Fixed

**Comment by spxtr** [comment]
ok

**Pull request merged**